docs/docs/APIdocs/Layers/Utilities/Input.md

```diff
@@ -0,0 +1,60 @@
+## Input ##
+
+**Scala:**
+```scala
+val input = Input()
+```
+**Python:**
+```python
+input = Input()
+```
+
+Input layer do nothing to the input tensors, just pass them. It should be used as input node
```

Comment:
It is used as input to the Graph container (add a link) when the first layer of the graph container ...